bsdploy/fabfile_digitalocean.py

```diff
@@ -33,20 +36,21 @@ def bootstrap(**kwargs):
     # overwrite sshd_config, because the DO version only contains defaults
     # and a line explicitly forbidding root to log in
     sudo("""echo 'PermitRootLogin without-password' > /etc/ssh/sshd_config""")
+    # additionally, make sure the root user is unlocked!
+    sudo('pw unlock root')
     sudo("""service sshd fastreload""")
     # revert back to root
     env.host_string = original_host
     # give sshd a chance to restart
     sleep(2)
     # clean up DO cloudinit leftovers
-    run("rm /etc/rc.d/digitalocean")
-    run("rm -r /etc/rc.digitalocean.d")
-    run("rm -r /usr/local/bsd-cloudinit/")
+    run("rm -f /etc/rc.d/digitalocean")
+    run("rm -rf /etc/rc.digitalocean.d")
+    run("rm -rf /usr/local/bsd-cloudinit/")
     run("pkg remove -y avahi-autoipd")
```

Comment:
this should also be put under an `if bu.os_release.startswith('10'):` - or else be made idempotent